src/inverseRobot.js

```diff
@@ -24,7 +24,18 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const invers = {};
+  const keys = Object.keys(robot);
+  const val = Object.values(robot);
+
+  for (let i = 0; i < keys.length; i++) {
```

Comment:
You can use for (const key of robot) cycle instead of creating two separate arrays with keys and values